packages/SwingSet/misc-tools/computrons-for-deliveries.py

```diff
@@ -0,0 +1,54 @@
+import sys, json, gzip, re, time
+from collections import defaultdict
+from pprint import pprint
+
+# Find all deliveries of a certain type and emit a histogram of how
+# many computrons they took. If the delivery is supposed to take
+# constant time, this might reveal JS-level growth (e.g. `serialize()`
+# taking longer because the data it is working with is growing).
+
+# You'll need to edit this file to change the selection criteria.
+
+fn = sys.argv[1]
+vatID = "v7"
+ko = None # "ko258"
+method = "serialize"
+
+crankNum = None
+
+print("crankNum,computrons")
```

Comment:
```suggestion
# Given a slogfile argument, emit a CSV for every matching delivery:
#   crankNum,computrons
# If the delivery is supposed to take constant time, this might reveal
# JS-level growth (e.g. `serialize()`  taking longer because the data it
# is working with is growing).

# These match criteria can be changed by hand:
vatID = "v7"
ko = None # "ko258"
method = "serialize"

print("crankNum,computrons")

crankNum = None
```